src/Servers/IIS/IIS/test/Common.FunctionalTests/StartupTests.cs

```diff
@@ -1006,9 +1006,14 @@ public async Task CheckStdoutWithLargeWrites_TestSink(string mode)
 
             StopServer();
 
-            var expectedString = new string('a', 30000);
-            Assert.Contains(TestSink.Writes, context => context.Message.Contains(expectedString));
-            EventLogHelpers.VerifyEventLogEvent(deploymentResult, EventLogHelpers.InProcessThreadExitStdOut(deploymentResult, "12", expectedString), Logger);
+            // Logs can be split due to the ANCM logging occuring at the same time as logs from the app, so check for a portion of the
+            // string instead of the entire string. The entire string will still be present in the event log.
+            var expectedLogString = new string('a', 16);
```

Comment:
Could this have the same issue as the 30k `a`'s, just a lot less likely?

Reply:
No, there are only a few log messages that can be spliced in between. So there is no way to split every string to a size smaller than 16.